docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/plugins-registration.md

```diff
@@ -77,21 +81,20 @@ class FileSystemDependencyProvider extends AbstractBundleDependencyProvider
     }
 }
 ```
-<em>Code example 1.2.2</em>: A single plugin registration with wrap method call
 
 ### 1.2.2. Plugins registration in an indexed array
-Manifests fully support multiple plugins registration in an indexed array. Manifests also support additional conditions for the registration of plugins, and additional restrictions on the order of plugins.
 
-Restrictions on the order of plugins can be done with special annotation keys ‘before' and ‘after’ (Code example 1.2.4). 
+Manifests fully support multiple plugins registration in an indexed array. Manifests also support additional conditions for plugin registration and restrictions on the order of the plugins.
```

Comment:
I believe order restrictions should be moved to a separate article, not necessarily in this space. This is a feature.

Reply:
It is useful to have all information related to the manifest at one page. 
Also, only one example with order is not enough for the new page

Can we left it here?

Reply:
for me it is ok. But we do not have a proper description of this feature. If I were a product developer I wouldn't understand this one.